lms/static/sass/shared/_alerts.scss

```diff
@@ -0,0 +1,169 @@
+// lms alerts
+// This file contains "I want alerts that look like the pattern library" sass
+// and should be replaced when moving to the patter library proper
+// https://github.com/edx/ux-pattern-library/blob/master/pattern-library/sass/patterns/_alerts.scss
+
+// ------------------------------
+// edX Pattern Library: Utilities - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// #CONFIG
+// #UTILITIES
+// #GENERAL
+// #INDIVIDUAL CASES
+
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$alert-background-color: #fcfcfc;
+$alert-icon-color: #fcfcfc;
+$alert-border-radius: 0.3125rem;
+// border around the sides and bottom
+$alert-border: 0.0625rem solid #cdd7db;
+
+// alert colors
+$alert-information-color: #6fa0ba;
+$alert-warning-color: #fdbc56;
+$alert-error-color: #b20610;
+$alert-success-color: #25b85a;
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+@mixin alert($alert-color) {
+    border-top:  rem(2) solid $alert-color;
+
+    .alert-icon {
+        color: $alert-icon-color;
+        background-color: $alert-color;
+    }
+}
+
+@mixin alert-message($width) {
+    @include float(left);
+    width: $width;
+    padding: 1.25rem;
+    padding-top: 0;
+    padding-bottom: 0;
+
+    :last-child {
+        // keeps the message compact
+        margin-bottom: 0;
+    }
+}
+
+// everything below here gets added specificity pattern-library-shim
+.pattern-library-shim {
```

Comment:
@AlasdairSwan, @andy-armstrong wanted me to loop you into this conversation re: sass files.

The issue is that we want the new alert I've created to appear as it would in the pattern library, but the actual pattern library isn't available. On Andy's recommendation I've copied most of `_alerts.scss` into this file, but I'm worried about re-defining the general `.alert` rule. How do you feel about wrapping the styles in `.pattern-library-shim` class as I've don?

Also, could one or both of you sanity-check the "backwards-compatible" pattern library rules I've defined here?

The big differences are:
- no ` @include susy-media`, I couldn't get these to work so I pulled the styles out of these blocks by default.
- no `palette()` calls for colors, I ended up getting the resolved values of these by inspecting http://ux.edx.org/components/alerts/
- similar logic applies to `spacing-vertical()` calls
- icon content changed to fontawesome unicode value
- `.icon` rules copied from https://github.com/edx/ux-pattern-library/blob/master/pattern-library/sass/patterns/_icons.scss#L143-L155
- my own addition for the placement of the div I'm adding (under `.subsection-header`)